README.md

```diff
@@ -83,10 +86,12 @@ a Portfolio page
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
+
 No live demo
 
 
 
+
```

Comment:
### GENERAL REVIEW FOR THE README FILE


- Hey, Please, kindly remove all placeholder text in the readme file. These placeholder texts are there to guide you on how to add the necessary content, so they're not expected to be in the readme after you customize it to suit your project needs. e.g 
> Describe 1 - 3 features you will add to the project.
---

- Please, kindly update this section in the image below, remove the headers that are not relevant to the project e.g (test), add some instructions on how users can have your project run locally on their machine, 

:point_down: 
![Screenshot from 2023-01-11 22-02-45](https://user-images.githubusercontent.com/59397159/211917195-0cd00c3d-863b-430c-84c2-ddb64186418d.png)
---
- Kindly tell users some features of the project so they can have an idea what the project is all about.
- Kindly add a license with a working link as this currently is broken to improve your readme file.
- kindly add some acknowledgment and some questions and answers about the project below:

:point_down: 
![Screenshot from 2023-01-11 22-08-22](https://user-images.githubusercontent.com/59397159/211918238-d26a689d-a90d-4be4-9e18-90bab3b69b20.png)